file_paths.md

```diff
@@ -252,6 +277,10 @@ user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        modules *                             # contains pxp-agent modules (ruby binaries)
+            pxp-module-puppet *               # executes Puppet and provides results of Puppet runs
```

Comment:
Just to be clear, `pxp-module-puppet` here is https://github.com/puppetlabs/pxp-module-puppet/blob/master/modules/puppet ? 

Double-checking since the filename is different.